docs/source/topics/performance.rst

```diff
@@ -0,0 +1,67 @@
+.. _topics:performance:
+
+***********
+Performance
+***********
+
+The performance of AiiDA depends on many factors:
+
+* the hardware that AiiDA is running on
+* the environment in which AiiDA is installed
```

Comment:
I had originally added this but already removed it when cherry-picking your commit, where you suggested to remove it. Think you were looking at an outdated diff.

What I mean with the environment is the runtime for AiiDA, i.e. is it installed with Conda, in virtual-env, or system wide, or maybe running in a container, etc. It is not quite clear if and how much of an impact it could have, but I thought it would be good to have the list complete. But now I think it is fine to just omit it for the time being.